polyfill.rs

```diff
@@ -0,0 +1,124 @@
+use alloc::string::String;
+use core::ffi::{c_int, c_uchar};
+use pink_extension::{error, info};
+use qjs_sys::c;
+use qjs_sys::convert::{js_val_from_bytes, js_val_into_bytes, js_val_into_u128};
+
+use crate::contract_call;
+
+trait IntoJsValue {
+    fn into_js_value(self, ctx: *mut c::JSContext) -> c::JSValue;
+}
+
+impl<T: AsRef<str>> IntoJsValue for Result<c::JSValue, T> {
+    fn into_js_value(self, ctx: *mut c::JSContext) -> c::JSValue {
```

Comment:
Brilliant. Let's document how the exception can be handled in js env.

Reply:
Exceptions are turned into js Exception, which can be caught in js code.
Will add some notes in the API docs.